samples/subsys/logging/kernel_event_logger/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _kerneleventlogger_sample:
+
+Kerneleventlogger Sample
+######################
+
+Overview
+********
+
+A simple application that demonstrates use of kernel event
+logger feature.Two threads (A and B) of same priority are
+created and another thread with less priority (i.e event_
+logger_thread)is also created which reads the events from
+kernel event loggerbuffer and prints on to the console.
```

Comment:
Change to:

     kernel event logger's buffer

to match what you've written below, and change "prints on to" to just "prints to"